src/dataframe/dataframe.jl

```diff
@@ -91,7 +91,9 @@ functions, or provide type assertions to the variables that hold columns
 extracted from a `DataFrame`.
 
 # Examples
-```julia
+```jldoctest
+julia> using DataFrames
```

Comment:
I think you should rather use https://juliadocs.github.io/Documenter.jl/stable/man/doctests/#Module-level-metadata for this.